packages/embeds/core/src/core.ts

```
@@ -71,13 +71,20 @@ class DubEmbed {
-        );
+      switch (event) {
+        case "ERROR":
+          onError?.(
+            new EmbedError({
+              code: data?.code ?? "",
+              message: data?.message ?? "",
+            }),
+          );
+          break;
+        case "PAGE_HEIGHT":
+          const container = document.getElementById(DUB_CONTAINER_ID);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)